ui/package.json

```diff
@@ -118,7 +118,7 @@
         "@wdio/sauce-service": "8.0.13",
         "@wdio/spec-reporter": "8.0.13",
         "babel-jest": "29.0.3",
-        "chromedriver": "108.0.0",
+        "chromedriver": "110.0.0",
```

Comment:
if there is a change in the package.json then the corresponding package-lock.json must also be included as part of the PR.

Reply:
updated `package-lock.json`! Is it supposed to introduce this many line diffs?

Reply:
hi @jimmytsang,
try to remove your node modules directory and run npm install.

Reply:
There are currently unwanted changes

Reply:
Thanks for the helpful feedback @mendi160 !
Ran the following commands `rm -rf node_modules; npm i`. How's it look now?

Reply:
There are still unwanted changes like these:
-                "qs": "^6.10.3",
+                "qs": "~6.5.2",
....
-                "got": "^12.1.0",
+                "got": "^8.3.1",

Please remove your package-lock.json as well and then run npm i

Reply:
done, please let me know if the file looks okay.

Reply:
now it looks ok. thanks